File: OdontoSysFrontEnd/OdontoSysModel/DetalleTratamiento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OdontoSysModel
{
    public class DetalleTratamiento
    {
        public int IdCita { get; set; }
        public Tratamiento Tratamiento { get; set; }
        public int Cantidad { get; set; }
        public double Subtotal { get; set; }

        public DetalleTratamiento()
        {
            this.Tratamiento = new Tratamiento();
        }

        public override string ToString()
        {
            return $"DetalleTratamiento{{IdCita={IdCita}, Tratamiento={Tratamiento}, Cantidad={Cantidad}, Subtotal={Subtotal}}}";
        }
    }
}

File: BackEnd/OdontoSys/OdontoSysModel/src/main/java/pe/pucp/edu/odontosys/services/model/DetalleTratamiento.java
package pe.pucp.edu.odontosys.services.model;


public class DetalleTratamiento {
    
    private Integer idCita;
    private Tratamiento tratamiento;
    private Integer cantidad;
    private double subtotal;

    public DetalleTratamiento() {
        this.tratamiento = new Tratamiento();
    }

    public Integer getIdCita() {
        return idCita;
    }

    public void setIdCita(Integer idCita) {
        this.idCita = idCita;
    }

    public Tratamiento getTratamiento() {
        return tratamiento;
    }

    public void setTratamiento(Tratamiento tratamiento) {
        this.tratamiento = tratamiento;
    }

    public Integer getCantidad() {
        return cantidad;
    }

    public void setCantidad(Integer cantidad) {
        this.cantidad = cantidad;
    }

    public double getSubtotal() {
        return subtotal;
    }

    public void setSubtotal(double subtotal) {
        this.subtotal = subtotal;
    }

    @Override
    public String toString() {
        String resultado = "";
        resultado += "================\n";
        resultado += "DetalleTratamiento\n";
        resultado += "----------------\n";
        resultado += "idCita: "       + idCita       + "\n";
        resultado += "tratamiento: "  + tratamiento  + "\n";
        resultado += "cantidad: "     + cantidad     + "\n";
        resultado += "subtotal: "     + subtotal     + "\n";
        resultado += "================\n";
        return resultado;
    }

}
